esmvaltool/utils/cmorizers/obs/cmor_config/OSI-409-nh.yml

```diff
@@ -0,0 +1,26 @@
+
+---
+# Common global attributes for Cmorizer output
+attributes:
+  dataset_id: OSI-409-nh
+  version: 'v1.2'
+  tier: 2
+  modeling_realm: reanaly
+  project_id: OBS
+  source: 'http://osisaf.met.no/p/ice/'
+  reference: 'osi-409'
+  comment: ''
+
+# Variables to cmorize (here use only filename prefix)
+variables:
+  sic:
+    mip: OImon
+    raw: ice_conc
+    grid: ease-125
+  # sic:
```

Comment:
Why the commented out stuff?

Reply:
First years have bidaily data but later it becomes daily. I have to do the last tests on Jasmin to see exactly when starts the daily data and then set-up the cmorization to generate monthly for the whole period and daily from that point to the end.